.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
 node_js:
 - 'stable'
+script:
+- npm run test
```

Comment:
Remove this, travis does this by default

Reply:
I've removed this